lib/ReactViews/Story/StoryPanel/StoryButtons.tsx

```diff
@@ -13,39 +15,38 @@ export const CollapseBtn = ({
 }: { isCollapsed: boolean } & BtnProp) => {
   const { t } = useTranslation();
   return (
-    <button
-      className={Styles.exitBtn}
+    <RawButton
       title={isCollapsed ? t("story.expand") : t("story.collapse")}
       onClick={onClick}
     >
-      <Icon glyph={isCollapsed ? Icon.GLYPHS.info : Icon.GLYPHS.arrowDown} />
-    </button>
+      {isCollapsed ? (
+        <StoryIcon styledWidth={"24px"} glyph={Icon.GLYPHS.info} />
+      ) : (
+        <StoryIcon styledWidth={"12px"} glyph={Icon.GLYPHS.arrowDown} />
+      )}
+    </RawButton>
   );
 };
 
 export const ExitBtn = ({ onClick }: BtnProp) => {
   const { t } = useTranslation();
+  const theme = useTheme();
   return (
-    <button
-      className={Styles.exitBtn}
-      title={t("story.exitBtn")}
-      onClick={onClick}
-    >
-      <Icon glyph={Icon.GLYPHS.close} />
-    </button>
+    <RawButton onClick={onClick} title={t("story.exitBtn")}>
+      <StoryIcon
+        styledWidth={"12px"}
+        glyph={Icon.GLYPHS.close}
+        css={`
+          border-radius: 50%;
+          border: 3px solid ${theme.textDark};
+          padding: 3px;
+        `}
```

Comment:
I can't remember original story panel design - do we want to keep the circle border around "X" close button here @philipgrimmett ?

### Before vs After

![image](https://user-images.githubusercontent.com/6187649/160762503-b1d35d20-9d49-4eac-8ee4-e2caf4d6f839.png)

Reply:
I took it from https://github.com/TerriaJS/terriajs/issues/5427#issuecomment-1032151750
https://www.figma.com/proto/0IrsjWUaA3QPYPNtDcoiNr/Terria---Stories---Compromise----Feb22?page-id=0%3A1&node-id=1%3A421&viewport=241%2C48%2C0.66&scaling=min-zoom&starting-point-node-id=1%3A421

Reply:
Ah cool! Sorry I forgot about that prototype.
Looks good!